### DISCUSSION TITLE
NextJS / ajax content

### DISCUSSION THREAD

**Discussion by redimongo** [vercel/next.js#27393] [Help] [answered]
I have asked this question on Stackoverflow with no success, so thought I would ask it here.
I need away to have the header and footer not move from the site no matter what page the user goes to - ajax style.

The reason is because in our current site we have a built in media player that plays while the user browsers the website.
If NextJS can't do this we can't make the switch to nextJS.

**Comment by miguderp** [comment, marked as answer]
Anything you place in `_app.js` gets mounted on the first render and never un-mounts even when you change page.
`_app.js` contains the shell, `index.js` is a page the same way `about.js` at the same level would be.

You can find a [basic example here](https://github.com/vercel/next.js/tree/canary/examples/with-app-layout)

**Reply by leerob** [reply]
Note: we've recently added Layouts docs!

https://nextjs.org/docs/basic-features/layouts
